### ISSUE TITLE
Persistent Item Robust Searching

### ISSUE THREAD

**Issue by Jacki3** [LoGaCulture/LUTE#55]
**Is your feature request related to a problem? Please describe.**
Inventory items require a specific folder structure and naming convention for persistence to work. Currently, we place inventory items into the 'Resources' folder and then into a new folder named 'Items'. 

**Describe the solution you'd like**
It would be ideal if we could simply search for objects of the base item type (they could still be placed in resource folder to make life easier and cleaner).

**Additional context**
Finding scriptable objects may not be easy as finding components such as Nodes and Orders but still should be doable - see how this is carried out for such items and apply similar logic.
